{"uuid": "a0335bee-adf8-4b29-8eb6-998f90974120", "children": ["2a20ab68-2280-43ce-ac8b-aaa53880ea0a", "275e6e0e-b90b-43fb-869a-68b7591d1c26", "1e2949c4-b703-489a-9e0b-d2c0eae7d1e2"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: DevToolsActivePort file doesn't exist\nStacktrace:\n0   chromedriver                        0x00000001092982c8 chromedriver + 4752072\n1   chromedriver                        0x0000000109218463 chromedriver + 4228195\n2   chromedriver                        0x0000000108e7bb18 chromedriver + 441112\n3   chromedriver                        0x0000000108ea707f chromedriver + 618623\n4   chromedriver                        0x0000000108ea2ddb chromedriver + 601563\n5   chromedriver                        0x0000000108e9ee3c chromedriver + 585276\n6   chromedriver                        0x0000000108ee1c43 chromedriver + 859203\n7   chromedriver                        0x0000000108ee174f chromedriver + 857935\n8   chromedriver                        0x0000000108ed9b33 chromedriver + 826163\n9   chromedriver                        0x0000000108eaa9fd chromedriver + 633341\n10  chromedriver                        0x0000000108eac051 chromedriver + 639057\n11  chromedriver                        0x000000010926530e chromedriver + 4543246\n12  chromedriver                        0x0000000109269a88 chromedriver + 4561544\n13  chromedriver                        0x00000001092716df chromedriver + 4593375\n14  chromedriver                        0x000000010926a8fa chromedriver + 4565242\n15  chromedriver                        0x00000001092402cf chromedriver + 4391631\n16  chromedriver                        0x00000001092895b8 chromedriver + 4691384\n17  chromedriver                        0x0000000109289739 chromedriver + 4691769\n18  chromedriver                        0x000000010929f81e chromedriver + 4782110\n19  libsystem_pthread.dylib             0x00007ff81b2424e1 _pthread_start + 125\n20  libsystem_pthread.dylib             0x00007ff81b23df6b thread_start + 15\n\n", "trace": "  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 900, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/runner/work/sauce_webtest/sauce_webtest/testcase/conftest.py\", line 22, in driver\n    get_driver = ChromeBrowser().browser\n  File \"/Users/runner/work/sauce_webtest/sauce_webtest/util/browser.py\", line 97, in browser\n    chrome = self._browser(service=self.service, options=self.options)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/chrome/webdriver.py\", line 81, in __init__\n    super().__init__(\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/chromium/webdriver.py\", line 106, in __init__\n    super().__init__(\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 288, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 381, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1668559552424, "stop": 1668559635818}], "start": 1668559552424, "stop": 1668559636475}